coldfront/core/project/templates/project/project_list.html

```diff
@@ -17,6 +27,13 @@
           Renew a PI's Allowance
         </a>
       {% endif %}
+      {% flag_enabled 'SECURE_DIRS_REQUESTABLE' as secure_dirs_requestable %}
```

Comment:
My first thought was to look for this in the project detail view.

Does it make sense to move this button there, so that the step of selecting a project in the form can be eliminated?

Reply:
We can do that. I assume this change will make the button will be less visible although this may not be important